tiledb/sm/query/readers/sparse_global_order_reader.cc

```diff
@@ -1874,117 +1878,38 @@ void SparseGlobalOrderReader<BitmapType>::process_slabs(
   std::sort(result_tiles.begin(), result_tiles.end(), result_tile_cmp);
 
   // Read a few attributes a a time.
-  uint64_t buffer_idx = 0;
+  std::optional<std::string> last_field_to_overflow{std::nullopt};
```

Comment:
NOTE TO REVIEWERS: Most of the deleted copy code here was moved as is in copy_tiles.